05_syntax_lab/03.py

```diff
@@ -5,3 +5,12 @@
 The result should be: 14
 """
 
+from random import randint
+num = randint(1,10000)
+print "num is:{}".format(num)
+sum = 0
+while num > 0:
+    digit = num % 10
+    sum += digit
```

Comment:
Note how github colors "sum" and "print" differently than "num". The reason is sum() is a function in python. Using it as a variable name works but can be confusing. `sum_digits` or `result` would be less confusing